docs/index.rst

```diff
@@ -1,58 +1,22 @@
 
-Welcome to Rikai's documentation!
-=================================
+Welcome to Rikai!
+=================
 
-Rikai is an `Apache Spark`_ based ML data format built for working with
-unstructured data at scale. Processing large amounts of data for ML is never trivial, but that
-is especially true for images and videos often at the core of deep learning applications. We are
-building Rikai with two main goals:
+Rikai is a large-scale data system specicially designed for Machine Learning workflows,
+specialized in Deep Learning development over various forms of unstructured data,
+for example, image, video or sensor data.
 
-1. Enable ML engineers/researchers to have a seamless workflow from feature engineering (`Spark`_)
-   to training (`PyTorch`_/`Tensorflow`_), from notebook to production (:doc:`See details <types>`).
-2. Enable advanced analytics capabilities to support much faster active learning, model debugging,
-   and monitoring in production.
+Rikai strives to offer agnostic developer experice to assist ML engineers at each
+stage of application development.
 
-.. code-block:: python
-
-  from pyspark.ml.linalg import DenseMetrix
-  from rikai.types import Image, Box2d
-  from rikai import numpy as np
-
-  df = spark.createDataFrame(
-      [{
-          "id": 1,
-          "mat": DenseMatrix(2, 2, range(4)),
-          "image": Image("s3://foo/bar/1.png"),
-          "annotations": [
-              {
-                  "label": "cat",
-                  "mask": np.random(size=(256,256)),
-                  "bbox": Box2d(xmin=1.0, ymin=2.0, xmax=3.0, ymax=4.0)
-              }
-          ]
-      }]
-  )
-
-  df.write.format("rikai").save("dataset/out")
-
-``Rikai`` dataset can be seamlessly integrated into your favorite training frameworks,
-taking `Pytorch`_ as an example:
-
-.. code-block:: python
-
-  from rikai.torch import DataLoader
-
-  data_loader = DataLoader(
-      "dataset/out",
-      shuffle=True,
-      batch=8,
-  )
-  for examples in data_loader:
-      print(example)
-
-Additionally, the parquet-native nature of the ``rikai`` format allows such unstructured ML dataset
-being analyzed in `Jupyter`_, `Spark`_, `Presto`_ or
-`BigQuery <https://cloud.google.com/bigquery/external-data-cloud-storage>`_.
+1. At its core, Riaki persists unstructured machine learning data in an `Apache Parquet`_
+   based format. It handles SerDe of these unstructured data transparently via a rich collection of
+   semantic types.
+2. Extensive set of I/O connectors, from ETL to training (i.e., `Pytorch`_ ``DataLoader``), to
+   bring the familiar developer experience at each stage of ML development.
+3. An SQL Engine, which extends `Spark`_ SQL with ML capability,
+   that analyzes Rikai data lake with your own model ("Bring Your Own Model").
+4. Carefully crafted data-visualization embedded with Semantic Types, especially in Jupyter notebooks.
```

Comment:
`Semantic Types` isn't really a specific thing, shouldn't be capitalized here?

Reply:
or did you mean to make this a link?

Reply:
ok, will change the case for now.